ui/src/components/DatePicker/index.jsx

```diff
@@ -15,6 +15,7 @@ import { date } from '../../utils/prop-types';
     '& > div': {
       alignItems: 'center',
     },
+    color: 'black',
```

Comment:
For colors to match the theme, you can use the color palette that we have. See https://github.com/taskcluster/taskcluster/search?q=theme.palette&unscoped_q=theme.palette.

Reply:
Alright. That's understood.

But changing the color property from here isn't really helping with fixing this issue. Right? 

Although, I now have the knowledge about theme colors for future use! So thanks!